src/GogeToken.sol

```diff
@@ -1298,20 +1246,21 @@ contract CakeDividendTracker is DividendPayingToken {
     event Claim(address indexed account, uint256 amount, bool indexed automatic);
 
     constructor() DividendPayingToken("DogeGaySon_Ethereum_Dividend_Tracker", "DogeGaySon_Ethereum_Dividend_Tracker", 0x0E09FaBB73Bd3Ade0a17ECC321fD13a19e81cE82) {
-    	claimWait = 3600;
-        minimumTokenBalanceForDividends = 200000 * (10**18); //must hold 10000+ tokens
+    	claimWait = 3600; // 1 hour
```

Comment:
Could replace 3600 with `1 hours` since Solidity has time units built in.

Reply:
As dumb as this sounds, I'm scared to touch anything in the CakeDividendTracker contract since it's legacy code I'm inheriting and works fine as of now. But you are correct

Reply:
Fair enough. If we have time we should deep dive into the dividends contract to really get an understanding of what's going on.